book.rb

```diff
@@ -0,0 +1,22 @@
+require './rentals'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] This should be together with the ```attr_accessor``` and not with the ```attr_reader```. Kindly make this change.